1-js/06-advanced-functions/02-rest-parameters-spread-operator/article.md

```diff
@@ -121,23 +121,23 @@ f(1); // 1
 ```
 ````
 
```

Comment:
Please,

- The code above seems to contain an error conflict. 
So, in such situations would you check the [English article](https://javascript.info/rest-parameters-spread-operator) (there is a link here) for ways to resolve the conflict?
From the text there, you might know how the Portuguese page would look.

Reply:
I didn't get this "error conflicts" at all, sorry. Can you explain me better what you mean by that?